src/plugins/workspace/public/components/workspace_switch/index.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useEffect } from 'react';
+import { EuiSelect, EuiTitle } from '@elastic/eui';
+import { useCallback } from 'react';
+import { useOpenSearchDashboards } from '../../../../../plugins/opensearch_dashboards_react/public';
+
+export const WorkspaceSwitch = () => {
```

Comment:
@raintygao With @SuZhou-Joe 's latest commit, switching workspace would simply be a url navigation, I think the workspace list can be simply a list of links, which navigate to url `?_workspace_id_=<workspace_id>`